examples/stm32f0/src/bin/multiprio.rs

```diff
@@ -126,6 +126,10 @@ fn main() -> ! {
     // Initialize and create handle for devicer peripherals
     let _p = embassy_stm32::init(Default::default());
 
+    // STM32s don’t have software-defined interrupts, so just use any free interrupt vectors which aren’t used
+    // by the rest of your application. In this case we’re using UART1 and UART2, but there’s nothing special
+    // about them. Any otherwise unused interrupt vector would work exactly the same.
```

Comment:
```suggestion
    // STM32s don’t have any interrupts exclusively for software use, but they can all be triggered by software as well as
    // by the peripheral, so we can just use any free interrupt vectors which aren’t used by the rest of your application.
    // In this case we’re using UART1 and UART2, but there’s nothing special about them. Any otherwise unused interrupt
    // vector would work exactly the same.
```

I'm not sure what a "software-defined" interrupt would be, but I think it's clearer to say that it doesn't have any interrupts reserved exclusively for software use (well, except PendSV and SysTick but they're core exceptions...), and note that any of the interrupts can be triggered by software.

Reply:
Sounds good, applied to all files